src/js/views/globals/modal/modal.hbs

```diff
@@ -12,6 +12,6 @@
 <div class="modal-footer" data-footer-region>
   <div class="u-float--right">
     <a href="#" class="modal-cancel js-close">{{ cancelText }}</a>
-    <button type="submit" class="{{ buttonClass }} js-submit">{{ submitText }}</button>
+    <button type="submit" class="button--primary {{ buttonClass }} js-submit">{{ submitText }}</button>
```

Comment:
I don't think `.button--primary` would go here.  If it's the better button than `button--blue` it should be the default, but `button--primary` is a specific style